fala/apps/adviser/tests/test_cookies_page_journeys_playwright.py

```diff
@@ -14,73 +14,147 @@ def test_cookies_page_view_accesed_from_banner(self):
 
 
 class CookiesPageEndToEndJourneysWithFreshSetUp(StaticLiveServerTestCase):
-    def test_that_cookie_has_appropriate_value(self):
+    banner_text = "Cookies on Find a legal adviser or family mediator"
+
+    def check_no_cookies(self, my_cookies):
+        assert len(my_cookies) == 0, "Cookies were found, which is surprising"
+
+    def verify_cookie_value(self, expected_value, my_cookies):
+        cookie_found = False
+        for cookie in my_cookies:
+            print(f"Name: {cookie['name']}, Value: {cookie['value']}")
```

Comment:
Hello print what brings you here. How strange, you're linter (pre-commit) should have caught that.

Reply:
In future if you feel there needs to be some type of logging for tests, I'd recommend using python logging and not print. Print is create for debugging.

Reply:
@psweeting1 could you give me an example of how to implement that? There isn't any specific for python logging in the playwright docs. All the stack overflow examples, would use [print](https://stackoverflow.com/questions/77945378/cant-capture-browser-error-logs-in-playright-test-w-python) statements.

Do you mean [this](https://realpython.com/python-logging/)?

Reply:
For the moment, I'll remove this.